### ISSUE TITLE
Translating Django auth models

### ISSUE THREAD

**Issue by sueli-rabaca** [deschler/django-modeltranslation#339]
I would like to know the best way to translate authorization models, like Group and Permission (standard way, meaning, not customized).

Modeltranslation works perfectly with my apps, but I don’t know how to do it for “third parties” (since this is a Django class - django.contrib.auth.models).

My environment is Python 3.4.3 and Django 1.8.4, and migration is running. django-modeltranlation==0.10.1

Thanks in advance.

**Comment by zlorf** [comment]
It should be as easy as this:
```python
from django.contrib.auth.models import Group
from modeltranslation.translator import translator, TranslationOptions

@register(Group)
class GroupTranslationOptions(TranslationOptions):
    fields = ('name',)  # or whatever these fields are called
```

**Comment by sueli-rabaca** [comment]
It was performed as suggested, but I received the following error:

    ValueError: Could not find manager NewMultilingualManager in modeltranslation.translator.
    Please note that you need to inherit from managers you dynamically generated with 'from_queryset()'.

    Process finished with exit code 1

I tried to follow the issue [issue #330](https://github.com/deschler/django-modeltranslation/issues/330), but it didn't work either. Any ideas?

**Comment by gabn88** [comment]
Same issue here. I put the modeltranslation on the Permission model of contrib.auth.models. Cannot update to Django 1.8 or higher due to this issue :(

**Comment by zlorf** [comment]
Does it show on Django 1.8 as well?

**Comment by gabn88** [comment]
yes, I tried Django 1.8.3 and Django 1.9, both gave this error. Django 1.7.9 is fine.

**Comment by zlorf** [comment]
Current RC should fix this.

**Comment by xelnod** [comment]
We have to migrate django.contrib.auth models in order to use that, hence run 'makemigrations' on test and production servers; is there no better way?

**Comment by roberfi** [comment]
I would like to rescue this topic. I want to translate some fields of a third party application and I've tried the following options:
* Create a `TranslationOptions` class with external model ➡ migrations are made in the third app's folder (inside venv)
* Create a wrapper app with Proxy models ➡ translatable fields are not created

Do you have any better idea to do so?

**Comment by last-partizan** [comment]
https://django-modeltranslation.readthedocs.io/en/latest/registration.html#committing-fields-to-database

You can try using `MIGRATION_MODULES`.

Copy original app migrations into folder in your project, and configure setting accordingly.

Be wary, when third-party app will add it's own migrations and you'll get conflicts this way. But they can be resolved, just be extra careful and make database backups.

**Comment by roberfi** [comment]
> https://django-modeltranslation.readthedocs.io/en/latest/registration.html#committing-fields-to-database
> 
> You can try using `MIGRATION_MODULES`.
> 
> Copy original app migrations into folder in your project, and configure setting accordingly.
> 
> Be wary, when third-party app will add it's own migrations and you'll get conflicts this way. But they can be resolved, just be extra careful and make database backups.

`MIGRATION_MODULES` just did what I wanted. I've added the following settings:
```python
MIGRATION_MODULES = {
    "my_app": "third_party.my_app.migrations",  # I had to create __init__.py file here
}
MODELTRANSLATION_TRANSLATION_FILES = (
    "third_party.my_app.translation",  # This is the translation.py file for the third party app
)
```

Then, I ran `python manage.py makemigrations myapp` and `python manage.py migrate myapp` and everything started working as expected!

Thank you so much.
